fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarController.kt

```diff
@@ -174,13 +177,13 @@ class DefaultBrowserToolbarController(
                 }
             }
             is TabCounterMenu.Item.NewTab -> {
-                activity.browsingModeManager.mode = BrowsingMode.Normal
+                appStore.dispatch(AppAction.ModeChange(BrowsingMode.Normal))
```

Comment:
Just a heads up that some form of this is still required, because this button doesn't actually create a new tab so the binding to the browser store won't cover it. This just navigates to the home screen in the expected mode.